stream_interfaces.go

```diff
@@ -30,6 +35,12 @@ type ServerStreamingClient[Res any] interface {
 // request, many responses) RPC. It is generic over the type of the response
 // message. It is used in generated code.
 type ServerStreamingServer[Res any] interface {
+	// Send sends a response message to the client. This method can be called
```

Comment:
@janardhanvissa can you provide the documentation for this text? I don't find all the text you have here https://grpc.io/docs/languages/go/generated-code/#server-streaming-methods.

There is lot of repetition here. Can you include relevant text only from above link and this https://github.com/grpc/grpc-go/issues/1894#issuecomment-653644035 ?

There is also some information here that we should include https://github.com/grpc/grpc-go/blob/master/Documentation/concurrency.md

Reply:
Please do this for all methods. Avoid repetition and only include text from official documentation

Reply:
For example the function docstring for `Send` can be simply this

```
Send can send a stream of messages to the client. It may be called multiple times to send multiple messages. Though one must avoid calling it multiple times against the same stream.
```

And the general information about end of stream and error can be moved to the interface docstring

```
If an error occurs on the stream, the error returned will be implemented by the 'status' package.  Please see the documentation in that package for more information. End-of-stream for the server-to-client stream is caused by the return of the handler method
```